### PR TITLE
Add tck delete

### PR DESCRIPTION
**Pull request by HarrisChu**

### CODE REVIEW

File: tests/tck/features/delete/DeleteVertex.IntVid.feature [hunk 1]
```diff
@@ -196,4 +196,23 @@ Feature: Delete int vid of vertex
       """
     Then the result should be, in any order:
       | VertexID | player.name | player.age |
+    When executing query:
```
Comment (Shylock-Hg) [inline tests/tck/features/delete/DeleteVertex.IntVid.feature:199]: Why don't add to `lookup` folder?

Comment (Shylock-Hg) [inline tests/tck/features/delete/DeleteVertex.IntVid.feature:199]: I think you could add it as a separate scenario.

**Comment by Shylock-Hg** [comment]
What't the issue?

**Comment by HarrisChu** [comment]
fixed https://github.com/vesoft-inc/nebula-graph/pull/917

closed the  pr

**Pull request closed without merging**